python/tank/util/__init__.py

```diff
@@ -22,6 +22,7 @@
     is_version_newer,
     is_version_older_or_equal,
     is_version_newer_or_equal,
+    suppress_known_deprecation,
```

Comment:
Again, not a blocker but I really wonder why bother listing all these modules, classes, and methods in this file instead of using Python's default import system? (`from utils import version` then `version.suppress_known_deprecation`)

Reply:
From what I've seen it's just to simplify the importing workflow. Maybe that was the original code style for tk-core.